contracts/src/v0.8/automation/2_0/KeeperRegistryBase2_0.sol

```diff
@@ -444,6 +445,22 @@ abstract contract KeeperRegistryBase2_0 is ConfirmedOwner, ExecutionPrevention {
     return transmitter.balance;
   }
 
+  function _blockNum() internal view returns (uint256) {
+    if (i_mode == Mode.ARBITRUM) {
+      return ARB_SYS.arbBlockNumber();
+    } else {
+      return block.number;
+    }
+  }
+
+  function _blockHash(uint256 blockNum) internal view returns (bytes32) {
+    if (i_mode == Mode.ARBITRUM) {
+      return ARB_SYS.arbBlockHash(blockNum);
```

Comment:
can we return zero here to keep consistent with blockhash() behaviour. Also a revert here will not result in a reorgedUpkeep log (ran into this during arb testing)